src/controllers/controlpickermenu.cpp

```diff
@@ -1633,7 +1633,7 @@ void ControlPickerMenu::addPlayerControl(const QString& control,
         const QString& controlDescription,
         QMenu* pMenu,
         bool deckControls,
-        bool deckStemControls,
+        [[maybe_unused]] bool deckStemControls,
```

Comment:
```suggestion
#ifdef __STEM__
        bool deckStemControls,
#endif
```

Reply:
This alone does not work. We need also apply the `#ifdef __STEM__` at the caller.
I like to avoid this clutter and keep the current solution.